src/release/breaking-changes/clipboard-data-required.md

```diff
@@ -0,0 +1,62 @@
+---
+title: At least one clipboard data variant must be provided
+description: In preparation for supporting multiple clipboard data variants, at least one clipboard data variant must be provided.
+---
+
+## Summary
+
+The [`ClipboardData constructor`][]'s `text` argument is no longer nullable.
+Code that provides `null` to the `text` argument must be migrated to provide
+an empty string `''`.
+
+## Context
+
+In preparation for supporting multiple clipboard data variants, the
+`ClipboardData` constructor now requires that at least one data variant is
+provided.
```

Comment:
I didn't mention this, but platforms were inconsistent in how they handled `null`. Some would reset the clipboard, some set the clipboard to the text `"null"`, others would crash. This was an unintended consequence of platforms assuming that at least one data variant is provided. We made the behavior consistent by disallowing this case.

Reply:
How about something like this:

"Previously, platforms were inconsistent in how they handled `null`. Behavior is now consistent across platforms. If you are interested in the low level details, see PR xxx."

I can imagine that some would want more info.

Reply:
Added!